feathr-ai/helm-charts
# Redis deployment on k8s

## Description

Added Redis on k8s

## How was this PR tested?


1. Run `helm install <Release Name> ./feathr-online`
2. After installation, go to AKS Portal and check for redis deployment. It should have the name `<Release Name>-feathr-online-redis`, then go to its Pod and copy the Pod name.
<img width="1377" alt="screen_shot 2023-02-14 at 15 43 49" src="https://user-images.githubusercontent.com/51813538/218659627-652e3899-edcb-441c-923d-5f4ee204e9ec.png">
4. Run `kubectl exec -it <Pod Name> -- redis-cli` and enter the Redis Pod.
5. Once you are in, make sure you can run `CONFIG GET maxmemory` to see if Redis is up and running.

## Does this PR introduce any user-facing changes?

- [ ] Yes. Make sure to clarify your proposed changes.
